src/models/User.ts

```diff
@@ -0,0 +1,6 @@
+export default interface User {
+    id: number,
+    login: string,
+    password: string,
+    refreshToken: string
```

Comment:
In my opinion, it would be better to store **access token** in the current user context instead of **refresh token** (which will be probably stored in http-only cookies)

Reply:
Згода, це лише початковий код, кукі ще не прикручував